charts/thoras/templates/dashboard/rbac.yaml

```diff
@@ -1,5 +1,22 @@
 {{- if and .Values.thorasDashboard.rbac.create .Values.thorasDashboard.enabled }}
 ---
+apiVersion: v1
```

Comment:
I moved this so the env field we're testing is in the same array index across all rbac.yaml files . This let's us to have one helm unit test instead of many and the env var ordering shouldn't matter